docs/users/cli-usage.md

```diff
@@ -29,6 +29,109 @@ Available Commands:
 Use "ddev [command] --help" for more information about a command.
 ```
 
+## Quickstart
+ddev is designed to be as simple as possible to incorporate into existing Wordpress and Drupal workflows. You can start using ddev with any site just by running a few commands.
+
+Below are quickstart instructions for each app type; Wordpress, Drupal 7, and Drupal 8.
+
+**Note:** If you do not have ddev already on your machine, please follow the [installation instructions](https://ddev.readthedocs.io/en/latest/#installation) before beginning the quickstart tutorial. 
```

Comment:
I *think* what you need to do with links is use a relative link rather than pointing to readthedocs. That way it won't be broken if things move around. So link would be ../index.md#installation I *think* (this fix should go wherever links are required; it also makes it work if people aren't on the readthedocs site)